libsolidity/formal/BMC.cpp

```diff
@@ -101,9 +101,9 @@ void BMC::analyze(SourceUnit const& _source, map<ASTNode const*, set<Verificatio
 	m_errorReporter.clear();
 }
 
-bool BMC::shouldInlineFunctionCall(FunctionCall const& _funCall, ContractDefinition const* _contract)
+bool BMC::shouldInlineFunctionCall(FunctionCall const& _funCall, ContractDefinition const* _scopeContract,  ContractDefinition const* _contextContract)
```

Comment:
Maybe this line is too long
```suggestion
bool BMC::shouldInlineFunctionCall(
	FunctionCall const& _funCall, 
	ContractDefinition const* _scopeContract,  
	ContractDefinition const* _contextContract
)
```